package-lock.json

```diff
@@ -4,6 +4,170 @@
   "lockfileVersion": 1,
```

Comment:
Jenkins complains that `package-lock.json` has parse error.
Could you re-create `package-lock.json` ?

Reply:
Fixed, was caused by the merge conflict I fixed.